packages/teleport-project-generator/src/index.ts

```diff
@@ -89,6 +91,15 @@ export const createProjectGenerator = (strategy: ProjectStrategy): ProjectGenera
 
       injectFilesToPath(rootFolder, path, files)
       collectedDependencies = { ...collectedDependencies, ...dependencies }
+
+      const { module } = strategy.pages.options || {
```

Comment:
this would be simplified if module would be placed next to generator, not inside options

Reply:
Yeah makes sense, but all projects don't need this. Since it is a flavor specific one, placed it under options

Reply:
yes, but it can be optional and not used by other projects.